references/depth/stereo/presets.py

```diff
@@ -0,0 +1,101 @@
+from typing import Tuple, Union
+
+import torch
+import transforms as T
+
+
+class StereoMatchingEvalPreset(torch.nn.Module):
+    def __init__(self, size=None) -> None:
+        super().__init__()
+
+        transforms = [
+            T.ToTensor(),
+            T.MakeValidDisparityMask(512),  # we keep this transform for API consistency
+            T.ConvertImageDtype(torch.float32),
+            T.Normalize(mean=0.5, std=0.5),
```

Comment:
I think it is better to put `mean` and `std` as part of the `__init__` param as well.